### PR TITLE
Fix bug with typespecs declared in package

### PR DESCRIPTION
**Pull request by alaindargelas**
Fixes issue #1859 which is a major bug.

### CODE REVIEW

File: frontends/systemverilog/uhdm_ast.cc [hunk 1]
```diff
@@ -4421,11 +4421,13 @@ void UhdmAst::process_logic_var()
     // TODO: add const attribute, but it seems it is little more
     // then just setting boolean value
     // current_node->is_const = vpi_get(vpiConstantVariable, obj_h);
+    bool anonymousTypespec = false;
```
Comment (kbieganski) [inline frontends/systemverilog/uhdm_ast.cc:4424]:
```suggestion
    bool anonymous_typespec = false;
```

**Review by kbieganski — CHANGES_REQUESTED** [review]
Looks OK, let's just stick to the naming convention.

**Comment by kgugala** [comment]
It seems there are conflicts here @alaindargelas can you rebase this PR so we can merge it?

**Comment by alaindargelas** [comment]
PR #2072 fixed this problem too.
So the code change in this PR is no longer needed (Actually does not work anymore as-is).
I removed the code change, this PR is only a new testcase to prevent future regression.

**Review by kgugala — APPROVED** [review]

**Pull request merged**